packages/client/modules/userDashboard/components/OrgBilling/DowngradeModal.tsx

```diff
@@ -0,0 +1,219 @@
+import styled from '@emotion/styled'
+import {Close} from '@mui/icons-material'
+import {Checkbox} from '@mui/material'
+import React, {useState} from 'react'
+import DialogContainer from '../../../../components/DialogContainer'
+import DialogContent from '../../../../components/DialogContent'
+import DialogTitle from '../../../../components/DialogTitle'
+import PlainButton from '../../../../components/PlainButton/PlainButton'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {TeamBenefits} from '../../../../utils/constants'
+import DowngradeToStarterMutation from '../../../../mutations/DowngradeToStarterMutation'
+import {useFragment} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import useMutationProps from '../../../../hooks/useMutationProps'
+import SendClientSegmentEventMutation from '../../../../mutations/SendClientSegmentEventMutation'
+import {DowngradeModal_organization$key} from '../../../../__generated__/DowngradeModal_organization.graphql'
+
+const StyledDialogContainer = styled(DialogContainer)({
+  padding: 8
+})
+
+const StyledDialogTitle = styled(DialogTitle)({
+  fontSize: 24,
+  lineHeight: '32px'
+})
+
+const StyledDialogContent = styled(DialogContent)({
+  display: 'flex',
+  flexDirection: 'column',
+  paddingBottom: 16
+})
+
+const LabelGroup = styled('div')({
+  paddingTop: '32px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  width: '100%'
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_700,
```

Comment:
this should be Slate 900, same as the labels below

Reply:
Sure thing!

Here's how the update looks:

![Screenshot 2023-03-01 at 18 20 02](https://user-images.githubusercontent.com/39854876/222228285-5aa85885-86d3-4367-923b-dd031d32a83c.png)

![Screenshot 2023-03-01 at 18 19 41](https://user-images.githubusercontent.com/39854876/222228414-ce123ead-88e0-4a95-9e6c-3cd3bc79fef7.png)